doc/api/crypto.md

```diff
@@ -944,6 +944,15 @@ The `output_format` can specify one of `'latin1'`, `'hex'` or `'base64'`. If
 `output_format` is provided a string is returned; otherwise a [`Buffer`][] is
 returned.
 
+The optional `options` argument is an object which specifies additional
+cryptographic parameters. Currently, the following options are supported:
+
+* `padding` : {String} - RSA padding, either `'pkcs1'` for RSASSA-PKCS1-v1_5
```

Comment:
Sure, but note that the rest of the document seems to use the uppercase variant and has a space before the colon.